app/src/main/java/lufra/youpidapp/Discotheque.kt

```diff
@@ -79,16 +89,24 @@ class Discotheque(private val context: Context) {
         return mp
     }
 
+    // par défault on utilise "all" et pas "musics"
     private fun playOne(name: String): Int {
+        return playOne(name, all)
+    }
+
+    /**
+     * Joue le son "name" en mode "un seul à la fois". soundList est soit "all" soit "musics"
+     */
+    private fun playOne(name: String, soundList: HashMap<String, Int>): Int {
```

Comment:
```suggestion
    /**
     * Joue le son "name" en mode "un seul à la fois".
     * @param name  le nom du son à jouer
     * @param soundList  une Map de nom de son (valides) vers un id de resource de son (valide).
     */
    private fun playOne(name: String, soundList: Map<String, Int> = all): Int {
```
En vrai il faudrait peut-être refactor ça et juste passer l'entier `soundList[name]!!`...

Reply:
En effet ça semble plus judicieux (j'avais oublié qu'en Kotlin on pouvait mettre des paramètres par défaut 😅 ). je vais refractor pour directement passer les id, ça semble plus logique que passer un hashmap